test/cxplat/lib/um/cxplat_um.vcxproj

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <ProjectGuid>{93edb3a0-c6e4-42cc-85a3-92bfc44964d0}</ProjectGuid>
+    <TargetName>cxplat_um</TargetName>
+    <UndockedType>lib</UndockedType>
+    <UndockedDir>$(SolutionDir)submodules\undocked\</UndockedDir>
+    <UndockedOut>$(SolutionDir)artifacts\</UndockedOut>
+    <UndockedSourceLink>true</UndockedSourceLink>
+  </PropertyGroup>
```

Comment:
Why don't you set to use the driver toolset? I think that will allow you to use the WPP stuff and not have to run the command yourself. Maybe it helps with the VCInstallDir stuff too?

Reply:
I attempted this and ran into some interesting linking issues with runtime libs that I could not resolve in a reasonable amount of time. So I think I will leave this be for now

Reply:
Sounds good. Thanks for trying.